frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature

```diff
@@ -73,11 +75,12 @@ Feature: Create the pipeline from builder page
              Then user will be redirected to Pipeline Details page with header name "<pipeline_name>"
 
         Examples:
+        #For git and image resource apply ../../testData/pipelines-workspaces/demo-optional-resources.yaml
                   | pipeline_name | task_name        | resource_type | resource_name |
                   | p-git         | openshift-client | Git           | git repo      |
                   | p-img         | buildah          | Image         | image repo    |
-                  | p-storage     | task-storage     | Storage       | storage repo  |
-                  | p-cluster     | task-cluster     | Cluster       | cluster repo  |
+                  # | p-storage     | task-storage     | Storage       | storage repo  |
+                  # | p-cluster     | task-cluster     | Cluster       | cluster repo  |
 
 
         @regression
```

Comment:
Is the epic number still required?

Reply:
Yes, to show the user still can go with these options but we lack appropriate test data